collector/exporter.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/alecthomas/kingpin/v2"
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
-	_ "github.com/go-sql-driver/mysql"
+	xmysql "github.com/go-sql-driver/mysql"
```

Comment:
I've added https://github.com/prometheus/mysqld_exporter/pull/738 to fix the name conflict.

Reply:
I have rebase from main branch, and roll backup to mysql.